### PR TITLE
Last Pull

### PR DESCRIPTION
**Pull request by namtn-2711**
- Last Pull

### CODE REVIEW

**Review by vietnh-1109 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -709,5 +709,298 @@ func renderLargeShapes(shapes: [Shape]) {
...
+```swift
+// MARK: Configure tableView
+
+private func configureTableView() {
+  // ...
+}
+
+// MARK: Action setup
+
+let actions =  {
```
Comment (tuanlvt-1117) [inline README.md]:
```suggestion
let actions = {
```

File: README.md [hunk 2]
```diff
@@ -709,5 +709,298 @@ func renderLargeShapes(shapes: [Shape]) {
...
+}
 ```
+
+**Tốt:**
+```swift
+private func configureTableView() {
+  // ...
+}
+
+let actions =  {
```
Comment (tuanlvt-1117) [inline README.md]:
```suggestion
let actions = {
```

**Review by tuanlvt-1117 — APPROVED** [review]

**Review by tuannm-1276 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -709,5 +709,298 @@ func renderLargeShapes(shapes: [Shape]) {
...
+**[⬆ Về đầu trang](#mục-lục)**
+
+### Các hàm gọi và hàm được gọi nên nằm gần nhau
+Nếu một hàm gọi một hàm  khác, hãy giữ những hàm này nằm gần theo chiều dọc trong
+file. Lí tưởng là, hãy giữ cho hàm gọi ở trên hàm được gọi. Chúng ta có xu hướng
+đọc code từ trên xuống, giống như đọc báo vậy. Do đó, hãy làm cho code của chúng
+ta cũng được đọc theo cách đó.
+
+**[⬆ Về đầu trang](#mục-lục)**
+
```
Comment (daolq-2712) [inline README.md]:
```suggestion
```
Remove em ei,
duplicate với ở trên [line 880](https://github.com/awesome-academy/clean-code-swift/pull/5/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R880)

**Review by daolq-2712 — CHANGES_REQUESTED** [review]
Pls check my comment!

**Review by daolq-2712 — APPROVED** [review]
LGTM!

**Pull request merged**